### PR TITLE
Add The Readme file for Anuket-Documentation Repository

### PR DESCRIPTION
**Pull request by Aryan-Singla**
Add The Readme file for the Anuket-Documentation Repository describes what the repository is all about

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+### Verify Python and Pip Installation
+
+Ensure Python and pip are installed and accessible in your environment.
+
+```bash
+python --version
+pip --version
+```
+
+### Install Sphinx
```
Comment (CsatariGergely) [inline Readme.md]: Sphinx and all other dependencies will be installed by tox. 

Here we should install only tox with `pip install tox`

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Install Sphinx using pip:
+
+```bash
+pip install sphinx
+```
+
+## Navigate to the Documentation Directory
```
Comment (CsatariGergely) [inline Readme.md]: This is not needed

File: Readme.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+```bash
+cd docs
+```
+
+## Building the Documentation
+
+To build the documentation locally, you can use the following commands. This example assumes you are using Sphinx:
+
+```bash
+python -m sphinx -b html . _build/html
```
Comment (CsatariGergely) [inline Readme.md]: Building should be done with `tox -e docs`

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

**Comment by Aryan-Singla** [comment]
Hi @CsatariGergely 
I have add your suggestions Please check

**Review by CsatariGergely — APPROVED** [review]

**Pull request merged**